cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Consider adding test cases to this file. You can start by testing your `Movie` model and `MovieSerializer` to ensure they behave as expected. Use Django's `TestCase` class to create unit tests for your application.